src/main/java/com/epam/page/object/generator/validators/ValidationExceptionConverter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.page.object.generator.validators;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.everit.json.schema.ValidationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ValidationExceptionConverter {
+
+    private final static Logger logger = LoggerFactory
+        .getLogger(ValidationExceptionConverter.class);
+
+    public List<ValidationResult> toValidationResult(ValidationException e) {
+
+        List<ValidationResult> validationResults = new ArrayList<>();
+        if (e.getCausingExceptions().isEmpty()) {
+            validationResults.add(getValidationResult(e));
+            logger.error(e.getMessage());
```

Comment:
logger should not log only message, entire stackTrace should be logged. 
If stacktraces are not welcome on error level, then keep message on error level, but add stacktrace on debug level.

Reply:
Done